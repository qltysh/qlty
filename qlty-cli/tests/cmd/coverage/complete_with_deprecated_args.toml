args = [
  "coverage",
  "complete",
  "--dry-run",
  "--override-commit-sha",
  "2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67",
  "--override-branch",
  "feature-branch",
  "--override-pr-number",
  "123",
  "--override-build-id",
  "build-789",
  "--override-commit-time",
  "2024-01-01T00:00:00Z",
  "--override-git-tag",
  "v1.0.0"
]
bin.name = "qlty"
status.code = 0

[env.add]
BUILDKITE_BUILD_NUMBER = "123"
BUILDKITE_BRANCH = "main"
BUILDKITE_BUILD_ID = "build-456"
QLTY_COVERAGE_TOKEN = "123"