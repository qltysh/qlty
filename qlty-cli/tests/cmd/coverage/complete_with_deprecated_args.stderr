qlty [..] [..]-[..] ([..])
[..]
https://qlty.sh/d/coverage

WARNING: --override-branch is deprecated and will be ignored

WARNING: --override-pr-number is deprecated and will be ignored

WARNING: --override-build-id is deprecated and will be ignored

WARNING: --override-commit-time is deprecated and will be ignored

WARNING: --override-git-tag is deprecated and will be ignored

 SETTINGS 

    cwd: [CWD]
    override-commit-sha: 2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67

 METADATA 

    Commit: 2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67

 AUTHENTICATION 

    Auth Method: Unknown
    Token: 123

 COMPLETING... 

    Coverage marked as complete in [..]s!

